src/html-media-source.js

```diff
@@ -264,4 +264,48 @@ export default class HtmlMediaSource extends videojs.EventTarget {
     this.sourceBuffers.push(buffer);
     return buffer;
   }
+
+  /**
+   * add a property to the VSB indicating the current audio/video/text stream
```

Comment:
I'd avoid acronyms (i.e. VSB) in our documentation or define them before using if it's really onerous.